tr/cs-221-states-models.md

```diff
@@ -0,0 +1,980 @@
+
+<br>
+
+
+**97. Game playing**
+
+&#10230; Oyun oynama
+
+<br>
+
+
+**98. In games (e.g. chess, backgammon, Go), other agents are present and need to be taken into account when constructing our policy.**
+
+&#10230; Oyunlarda (örneğin satranç, tavla, Go), başka oyuncular vardır ve politikamızı oluştururken göz önünde bulundurulması gerekir.
+
+<br>
+
+
+**99. Game tree ― A game tree is a tree that describes the possibilities of a game. In particular, each node is a decision point for a player and each root-to-leaf path is a possible outcome of the game.**
+
+&#10230; Oyun ağacı ― Oyun ağacı, bir oyunun olasılıklarını tarif eden bir ağaçtır. Özellikle, her bir düğüm, oyuncu için bir karar noktasıdır ve her bir kökten (root) yaprağa (leaf) giden yol oyunun olası bir sonucudur.
+
+<br>
+
+
+**100. [Two-player zero-sum game ― It is a game where each state is fully observed and such that players take turns. It is defined with:, a starting state sstart, possible actions Actions(s) from state s, successors Succ(s,a) from states s with actions a, whether an end state was reached IsEnd(s), the agent's utility Utility(s) at end state s, the player Player(s) who controls state s]**
+
+&#10230; [İki oyunculu sıfır toplamlı oyun ― Her durumun tamamen gözlendiği ve oyuncuların sırayla oynadığı bir oyundur. Aşağıdaki gibi tanımlanır:, bir başlangıç durumu sstart, s durumunda gerçekleştirilebilecek olası eylemler Actions(s), s durumunda a eylemi gerçekleştirildiğindeki ardıllar Succ(s,a), bir bitiş durumuna ulaşılıp ulaşılmadığı IsEnd(s), s bitiş durumunda etmenin elde ettiği fayda Utility(s), s durumunu kontrol eden oyuncu Player(s)]
+
+<br>
+
+
+**101. Remark: we will assume that the utility of the agent has the opposite sign of the one of the opponent.**
+
+&#10230; Not: Oyuncu faydasının işaretinin, rakibinin faydasının tersi olacağını varsayacağız.
+
+<br>
+
+
+**102. [Types of policies ― There are two types of policies:, Deterministic policies, noted πp(s), which are actions that player p takes in state s., Stochastic policies, noted πp(s,a)∈[0,1], which are probabilities that player p takes action a in state s.]**
+
+&#10230; [Politika türleri ― İki tane politika türü vardır:, Belirlenimci politikalar, πp(s) olarak gösterilir, p oyuncusunun s durumunda gerçekleştirdiği eylemler., Olasılıksal politikalar, πp(s,a)∈[0,1] olarak gösterilir, p oyuncusunun s durumunda a eylemini gerçekleştirme olasılıkları.]
+
+<br>
+
+
+**103. Expectimax ― For a given state s, the expectimax value Vexptmax(s) is the maximum expected utility of any agent policy when playing with respect to a fixed and known opponent policy πopp. It is computed as follows:**
+
+&#10230; En yüksek beklenen değer(Expectimax) ― Belirli bir s durumu için, en yüksek beklenen değer, Vexptmax(s), sabit ve bilinen bir rakip politikasına,πopp, göre oynarken, bir oyuncu politikasının en yüksek beklenen faydasıdır. En yüksek beklenen değer(Expectimax) aşağıdaki gibi hesaplanmaktadır:
```

Comment:
En yüksek beklenen değer(Expectimax) ― Belirli bir s durumu için, en yüksek beklenen değer olan Vexptmax(s), sabit ve bilinen bir rakip politikası olan πopp'a göre oynarken, bir oyuncu politikasının en yüksek beklenen faydasıdır. En yüksek beklenen değer(Expectimax) aşağıdaki gibi hesaplanmaktadır:

Reply:
OK